### PR TITLE
Uploads

### PR DESCRIPTION
**Pull request by cdoyle-temple**
Person photo now working.

### CODE REVIEW

File: app/controllers/admin/groups_controller.rb [hunk 1]
```diff
@@ -52,5 +52,12 @@ def update
 
     # See https://administrate-prototype.herokuapp.com/customizing_controller_actions
     # for more information
+
+    def update
```
Comment (nomadicoder) [inline app/controllers/admin/groups_controller.rb]: Do we need this? If not, we should remove debug override methods.

File: app/controllers/admin/people_controller.rb [hunk 2]
```diff
@@ -17,5 +17,62 @@ class PeopleController < Admin::ApplicationController
 
     # See https://administrate-prototype.herokuapp.com/customizing_controller_actions
     # for more information
+
```
Comment (nomadicoder) [inline app/controllers/admin/people_controller.rb]: Remove the commented out example code.

**Comment by cdoyle-temple** [comment]
Cleaned up the controllers.

**Comment by nomadicoder** [comment]
Thank you, looks good!

**Pull request merged**